### PR TITLE
initial UI for bridge page

### PR DESCRIPTION
**Pull request by ShawTim**

### CODE REVIEW

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -57,6 +57,16 @@ const Header = () => {
           </Typography>
         </Link>
       </div>,
+      <div key="navitem-savings" className="raft__header__nav-item">
```
Comment (lazarmitic) [inline src/components/Header/Header.tsx:60]: Key should be `navitem-bridge`.

File: src/components/Bridge/PoweredBy.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
+import { memo } from 'react';
+
+const RaftLogo = () => {
```
Comment (lazarmitic) [inline src/components/Bridge/PoweredBy.tsx:3]: Component name should be `PoweredBy`.

File: src/components/Bridge/Bridge.tsx [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+    () =>
+      formatCurrency(toBalanceAfter, {
+        currency: R_TOKEN,
+        fractionDigits: USD_UI_PRECISION,
+      }),
+    [toBalanceAfter],
+  );
+
+  const onSwapNetwork = useCallback(() => {
+    setFromNetwork(toNetwork);
```
Comment (lazarmitic) [inline src/components/Bridge/Bridge.tsx:55]: There is a chance we have a race condition here.

I would store toNetwork and fromNetwork in new variables, and then update state using these variables.
```
const fromNetworkCurrent = fromNetwork;
const toNetworkCurrent = toNetwork;

setFromNetwork(toNetworkCurrent);
setToNetwork(fromNetworkCurrent);
```

File: src/components/Bridge/PoweredBy.tsx (hunk 2 above)
Comment (ShawTim) [reply src/components/Bridge/PoweredBy.tsx]: resolved.

File: src/components/Bridge/Bridge.tsx (hunk 3 above)
Comment (ShawTim) [reply src/components/Bridge/Bridge.tsx:60]: as discussed, race condition wont happen in this case. leave it as it is.

**Review by lazarmitic — APPROVED** [review]

**Pull request merged**